incremental-search/search-suggestions/README-benchmarking.md

```diff
@@ -0,0 +1,43 @@
+# Benchmarcking results
+
+This is a comparison of the benchmarking results for indexed prefix search with fast-search and streaming prefix search. The benchmark tests were run with one query and a variable number of clients. Both benchmarkings used the same corpus containing 80 000 documents, with the documents being one to two terms long. The benchmarking was run with a duration of 60 seconds.
```

Comment:
Using linebreaks makes it easier to diff / review code later (but not very important)
```suggestion
This is a comparison of the benchmarking results for indexed prefix search with fast-search and streaming prefix search.
The benchmark tests were run with one query and a variable number of clients.
Both benchmarking runs used the same corpus containing 80 000 documents,
with the documents being one to two terms long.
The benchmarking was run with a duration of 60 seconds.
```